testapi.pm

```diff
@@ -905,7 +905,7 @@ C<$fail_message> is returned in the die message if specified.
 I<The C<script_run> implementation is distribution specific and not always available.
 For this to work correctly, it must return 0 if and only if C<$command> completes
 successfully. It must NOT return 0 if C<$command> times out. The default implementation
-should work on *nix operating systems with a configured serial device.>
+should work on *nix operating systems with a configured serial device.
```

Comment:
The `>` ends the `I<` from 4 lines above

Reply:
It seems to be a bug in Pod::Parser, which is used by Pod::AsciiDoctor.
pod2html is using Pod::HTML instead.
The bug happens for every `I<...>`.
I guess we have to live with it for now. I tried to dig through the source, and wasn't able to find where the parsing happens exactly so far.

Reply:
Do you think we should report an upstream issue about that or is there one already?